registry/protocol/protocol.go

```diff
@@ -21,23 +21,21 @@ import (
 	"context"
 	"strings"
 	"sync"
-)
```

Comment:
recover all import style in this file

Reply:
@zouyx  Hi, I still not do the changes yet. Will do later.

Reply:
@zouyx  hey, it's done now.